Controllers/DoctorsController.cs

```diff
@@ -53,7 +53,7 @@ public IActionResult Create()
         // more details, see http://go.microsoft.com/fwlink/?LinkId=317598.
         [HttpPost]
         [ValidateAntiForgeryToken]
-        public async Task<IActionResult> Create([Bind("DoctorId,DoctorName,DoctorAddress,DoctorAge,RMPNumber,HighestQualification,Speciality")] Doctor doctor)
+        public async Task<IActionResult> Create([Bind("DoctorId,DoctorName,DoctorAddress,DoctorAge,RMPNumber,HighestQualification,Speciality,CreatedBy,CreateDate,IsActive,ModifiedBy,ModifiedDate")] Doctor doctor)
```

Comment:
What if a model is having more then 50 properties, Is this right way to bind them?